src/lib/vrf_lib/standalone.ml

```diff
@@ -33,7 +40,7 @@ module Make
 
       val negate : t -> t
 
-      val scale : t -> Scalar.t -> t
+      val scale : t -> Scalar.Stable.V1.t -> t
```

Comment:
And then these (and all functions) should only use on `Scalar.t` (the latest value) and not directly get a stable type.